TestIntMem.java

```diff
@@ -0,0 +1,140 @@
+import java.util.Arrays;
```

Comment:
Copyright header missing.

Reply:
This is a draft PR, not yet opened for review. These tests are for my testing during development and may not be checked in for the final review. The tests will be modified, and the final version will have an appropriate copyright header.

Reply:
Please don't bother responding to our suggestions on Draft PR, the idea here is to educate you so that you can incorporate reviewer's suggestion in next version.